eng/common/mcp/README.md

```diff
@@ -0,0 +1,57 @@
+# Azure SDK MCP Servers
```

Comment:
Do we really want this in eng/common? I was expecting all our mcp tools would like live in the azure-sdk-tools repo itself.

Reply:
Just the script to run them and install mcp.json into vscode will live in common. 

Also FYI I'm revamping this a bit in favor of stdio mcp servers.